src/screens/accessCodeView/accessCodeViewStyle.ts

```diff
@@ -0,0 +1,156 @@
+import { styled } from '@mui/material';
+import { Avatar, Box, Button, Typography, Card, Input } from '@mui/material';
+
+import { theme } from '../../assets/styles/theme';
+
+export const Heading = styled(Typography)`
+  width: 80vw;
+  font-family: Inter, sans-serif;
+  font-size: 7vw;
+  font-weight: 400;
+  line-height: 6rem;
+  text-align: left;
+  margin: 10vh 1vw 4vh 1vw;
+  @media (max-width: 600px) {
+    font-size: 6vw;
+    margin: 4vh 2vw;
+    line-height: 2.42rem;
+  }
+`;
+
+export const SearchBar = styled(Input)`
+  border-bottom: 0.125rem solid ${theme.colors.grey_25};
+  width: 80vw;
+  height: 5vh;
+  margin: 2vh auto;
+  font-size: 4vh;
+  & .MuiInputBase-input::placeholder {
+    color: ${theme.colors.black_100};
+    opacity: 1;
+  }
+  @media (max-width: 600px) {
```

Comment:
`@media (max-width: 600px)` nie jest konieczne bo jest to warunek żeby zmiany zostały wykonane dla urządzeń o szerokości ekranu mniejszej niż 600px a jako że robimy tylko ekran dla aplikacji mobilnej nie będzie to puki co konieczne